### ISSUE TITLE
Need to validate input data

### ISSUE THREAD

**Issue by mirajhamid** [kana1/node-playground#12]
When we try to add data in text format. The system tries to save 'undefined' data into PollsDB without validating them. How ever system sends success response to the User. Need to validate only to save JSON data.

**Comment by kana1** [comment]
yes. It seems we are not validating any of the inputs send to the application. we need to improve the application to handle invalid data.
